dcos/dcos-ui
# DCOS-46265: Can't open Configuration for a specific service.

Change the format of the value passed to DateFormat.

Closes https://jira.mesosphere.com/browse/DCOS-46265

## Testing
1. Start a new single-container service (the parameters shouldn't matter)
2. Go to it's configuration screen
3. Verify that it works and doesn't show a black page with console errors
4. Verify that the versions dropdown menu works

## Trade-offs
None.

## Dependencies
None.

## Screenshots
### Before
![screenshot from 2019-02-21 16-03-40](https://user-images.githubusercontent.com/40791275/53175160-1446fe00-35f4-11e9-94c5-a3984bc94467.png)

### After
![screenshot from 2019-02-21 16-04-46](https://user-images.githubusercontent.com/40791275/53175168-190bb200-35f4-11e9-8e5e-16fbd4485f7d.png)
